pkg/apis/manager/v1beta1/api.proto

```diff
@@ -101,11 +101,11 @@ enum ParameterType {
  * Distribution types for HyperParameter.
  */
 enum Distribution {
-    UNIFORM = 0;
-    LOG_UNIFORM = 1;
-    NORMAL = 2;
-    LOG_NORMAL = 3;
-    DISTRIBUTION_UNKNOWN = 4;
+    DISTRIBUTION_UNKNOWN = 0;
```

Comment:
```suggestion
    DISTRIBUTION _UNSPECIFIED = 0;
```

I would like to select the `UNSPECIFIED` suffix here.
Please see: https://google.aip.dev/126

Reply:
Make sense, @tenzen-y should we rename other gRPC parameters to UNSPECIFIED ?

Reply:
Changing released gRPC, it indicates losing backward compatibility.
So, I would like to keep using the existing API for released gRPC, WDYT?

Reply:
Since these gRPC API is not exposed to the end-users, do you still think that we should not change the existing APIs ?
It only affects users who build their own Suggestion service.

Reply:
> Since these gRPC APIs are not exposed to the end-users, do you still think that we should not change the existing APIs ?

Almost correct. Additionally, when users keep using the removed Suggestion Services like the Chocolate Suggestion, users face the same problem.

So, can we collect feedback on the dedicated issue outside of here?

Reply:
SGTM

Reply:
> Sure, let's followup on this in the issue, and rename it after few months if we don't get any feedback. @shashank-iitbhu Please can you create an issue to track it ?

Sure, I will create a separate issue to track the renaming of other gRPC parameters to UNSPECIFIED.